src/hgvs/easy.py

```diff
@@ -66,6 +67,8 @@
 t_to_g = projector.t_to_g
 t_to_p = projector.t_to_p
 get_relevant_transcripts = am38.relevant_transcripts
+pretty37 = PrettyPrint(hdp, am37, useColor=True, showLegend=True)
```

Comment:
Please use snake case for consistency with rest of code base.

Reply:
done